src/System.Private.Xml/src/System/Xml/Base64Encoder.cs

```diff
@@ -13,7 +13,7 @@ internal abstract partial class Base64Encoder
         private int _leftOverBytesCount;
         private char[] _charsLine;
 
-        internal const int Base64LineSize = 76;
+        internal const int Base64LineSize = 1024;
```

Comment:
That rfc is only applicable to base64 encoded email as mail servers used to have a requirement that each line is no longer than 80 characters (hold over from terminals which were 80x25 I believe).

---
In reply to: [123563223](https://github.com/dotnet/corefx/pull/21155#discussion_r123563223) [](ancestors = 123563223)